templates/workinggremlindb.txt

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
What is this file?

Reply:
This was just a dummy test file I was working on which I have removed.